README.md

```diff
@@ -44,6 +44,77 @@ This is the basic outline of the project and is a work in progress.
 
 ## Usage Examples
 
+# Public auth example for RequestHandler
+```php
+$config [
+    'key' => CONSUMER_KEY,
+    'secret' => CONSUMER_SECRET,
+    'sign_with' => 'HMAC-SHA1',
+    // other options
+]
+
+// Get Request Token
+$handler = new RequestHandler($config);
+try {
+    $handler->request('GET', 'Currencies');
+} catch (AuthException $e) {
+    $authData = $handler->getAuthData();
+    // we store oauth_token from $authData and direct user to
+    // https://api.xero.com/oauth/Authorize?oauth_token=REQUEST_TOKEN_HERE
+}
+
+// Get Access Token
+// We assume that user granted access by visiting this url
+// https://app.xero.com/oauth/APIAuthorise?oauth_token=REQUEST_TOKEN_HERE
+$config['token'] = REQUEST_TOKEN_HERE;
+$config['token_secret'] = $authData['oauth_token_secret']; // see above
+$config['oauth_verifier'] = '563499'; // we collect after user grants permission
+$handler = new RequestHandler($config);
+$handler->request('GET', 'Currencies');
+$authData = $handler->getAuthData();
```

Comment:
Based on the great discussion on slack we've agreed that this is outide of this packages realm of responsibility.